Assets/LoggingManager/LoggingManager.cs

```diff
@@ -126,220 +95,147 @@ public void SaveLog(string collectionLabel)
         }
     }
 
+    public void SaveAllLogs()
+    {
+        foreach (KeyValuePair<string, LogStore> pair in logsList)
+        {
+            SaveLog(pair.Key);
+        }
+    }
+
+
     public void SetEmail(string newEmail)
     {
         email = newEmail;
     }
 
     public void CreateLog(string collectionLabel)
     {
-        collections.Add(collectionLabel, new LogCollection());
+        if (collectionLabel == "Meta")
+        {
+            AddMetaCollectionToList();
+            return;
+        }
+        LogStore logStore = new LogStore(collectionLabel, email, sessionID, logStringOverTime);
+        logsList.Add(collectionLabel, logStore);
     }
 
-    public void Log(string collectionLabel, Dictionary<string, object> logData, LogMode logMode = LogMode.Append)
+
+    public void Log(string collectionLabel, Dictionary<string, object> logData)
     {
-        if (!collections.ContainsKey(collectionLabel))
+        if (!logsList.ContainsKey(collectionLabel))
         {
-            collections.Add(collectionLabel, new LogCollection());
-            collections[collectionLabel].label = collectionLabel;
-        }
-        foreach (KeyValuePair<string, object> pair in logData)
-        {
-            if (!collections[collectionLabel].log.ContainsKey(pair.Key))
+            if (collectionLabel == "Meta")
```

Comment:
Again, the collectionLabel should ideally not have some magic words. Can't we just use the same code path as any other log? (given that LogStore holds info about logging behavior already)